features/page_objects/back_office/new/registration_edit_pay_by_bank_transfer_page.rb

```diff
@@ -0,0 +1,5 @@
+class RegistrationEditPayByBankTransferPage < SitePrism::Page
```

Comment:
You could avoid creating this page by using `@journey.standard_page` which contains standard items for a simple page with a heading, content and button.